CONTRIBUTING.md

```diff
@@ -66,6 +66,8 @@ Regularly, a new release of [mdn-browser-compat-data](https://www.npmjs.com/pack
  This step will trigger Travis to publish to npm automatically (see our [.travis.yml file](https://github.com/mdn/browser-compat-data/blob/master/.travis.yml)).
  6. Check [Travis CI](https://travis-ci.org/mdn/browser-compat-data) again for the v0.0.43 build and also check [mdn-browser-compat-data on npm](https://www.npmjs.com/package/mdn-browser-compat-data) to see if `0.0.43` shows up correctly once Travis has finished its work.
  7. Notify the [#mdndev](irc://irc.mozilla.org/mdndev) IRC channel on irc.mozilla.org about the new release and coordinate with jwhitlock or rjohnson a deployment of the new package to the MDN site.
+ 8. Create a [new release on GitHub](https://github.com/mdn/browser-compat-data/releases) and write down release notes. Mention notable changes if non-data updates happened (schema, test, or infrastructure changes). Also record current statistics such as total number of contributors and stargazers, as well as the total number of features by running this node command in the main dir:
```

Comment:
Can I suggest putting in a template in this file? Something like this, maybe?

```
Notable changes:
- [TK text]

Statistics:
- [TK number] contributors have changed [TK number] files with [TK number] additions and [TK number] deletions in [TK number] commits [TK link to diff between version tags]
- [TK number] total contributors
- [TK number] total stargazers
```

Not strictly necessary, just thought it would be a nice-to-have (assuming GitHub doesn't support release templates—I checked and it didn't look like it, but I would love to be corrected on this point)

Reply:
I've added this. I think the release template is a fantastic idea, and I wonder where the place was to submit ideas to GitHub.

Reply:
I poked around and apparently they ask people to use [their contact form](https://github.com/contact) to make feature requests. It might be shouting into the void, but I did send them a little note suggesting it (comparing it favorably to saved replies and issue templates).

Reply:
OK, an interesting update on this release templates question, @Elchi3. I actually got a reply from GitHub support (I did not expect nor ask for one). Apparently there's a set of query parameters for [automating releases](https://help.github.com/articles/automation-for-release-forms-with-query-parameters/). Thus [this link](https://github.com/mdn/browser-compat-data/releases/new?body=Notable+changes%3A%0D%0A-+%5BTK+text%5D%0D%0A%0D%0AStatistics%3A%0D%0A-+%5BTK+number%5D+contributors+have+changed+%5BTK+number%5D+files+with+%5BTK+number%5D+additions+and+%5BTK+number%5D+deletions+in+%5BTK+number%5D+commits+%5BTK+link+to+diff+between+version+tags%5D%0D%0A-+%5BTK+number%5D+total+contributors%0D%0A-+%5BTK+number%5D+total+stargazers%0D%0A-+%5BTK+number%5D+total+features) will open the page for a release prepopulated with the template text. It might be worth creating a script to generate the text outright.

Reply:
Ha! Clever! The more streamlined the release procedures are, more often we can release I think, so I think it's worth doing that. PR or issue is welcome on this :)